votor/src/consensus_pool.rs

```diff
@@ -402,6 +410,17 @@ impl ConsensusPool {
             "Validator stake is zero for pubkey: {validator_vote_key}"
         );
 
+        match self
```

Comment:
@qkniep : It seems like if I move this recording a few lines below after the check for the conflict votes, then we will not have the issue you noted of duplicate votes.  What do you think, should I move it?